### DISCUSSION TITLE
Unable to Find Output Simulation Files

### DISCUSSION THREAD

**Discussion by zixingmun** [LAPKB/Pmetrics#209] [Q&A]
Hi All,

I'm using Pmetrics for pharmacometrics modelling for the first time and I think I've successfully run simulations using `PM_sim$new`. 

It says that 
> The following file was successfully parsed: simout1.txt

and I have it to simout10.txt

It has also suggested that

> Use, for example, SIMparse(“simout1.txt”) to extract simulator output for analysis or plotting.

However, this is the error message I'm getting

> SIMparse("simout1.txt")
Error in SIMparse("simout1.txt") : No files found.

I was wondering if someone could help me out. Thank you!

**Comment by Siel** [comment]
Hi @zixingmun Thanks for using Pmetrics and reaching out.

`PM_sim$new` is an R6 method, it should return you back a simulation object than then you can interact with, see [here](https://lapkb.github.io/Pmetrics/reference/PM_sim.html).

On the other hand `SIMparse` is an S3 method intended to be used with `SIMrun`.

The correct way of doing it would be:

`my_sim <- PM_sim$new(...)`

In this case `my_sim` will be a `PM_sim` object. Then you could do things like:

`my_sim$plot()`
or
`my_sim$pta()`

Again, please check the documentation for the whole list of methods available to `PM_sim` objects.
